src/os_execd/win_execd.c

```diff
@@ -25,6 +25,31 @@
 OSList *timeout_list;
 OSListNode *timeout_node;
 
+/* Shut down win-execd properly */
+static void WinExecd_Shutdown(int signal)
```

Comment:
Functions registered in `atexit()` must get nothing and return nothing. Please remove `int signal` from the parameter list.